### PR TITLE
recover from intermittent auth flow fails; fix broken image paths

### PR DESCRIPTION
**Pull request by the-daniel-rothig**
### Context

Now that we return status codes on the original route, a lot of the errors we've seen have been diagnosed and have relatively easy fixes.

### Changes proposed in this pull request

- fix up some image references in _Layout_base.cshtml - those have pointed to nonexistent images causing 404s
- remove the check for "/auth/cb/" - this path is not registered by the OIDC middelware and that bit of code didn't do anything. DfE Sign in are fixing the trailing slash on their end today
- add a handler for remote failure. The most common of these is a "Correlation failed" exception. [This](https://github.com/aspnet/Security/issues/1165) suggests that this is due to load balancing and the recommendation is to just restart the auth process, which we now do.

### CODE REVIEW

File: src/ui/Startup.cs [hunk 1]
```diff
@@ -183,7 +183,7 @@ public void ConfigureServices(IServiceCollection services)
                     OnMessageReceived = context =>
                         {
                             var isSpuriousAuthCbRequest =
-                                (context.Request.Path == new Microsoft.AspNetCore.Http.PathString("/auth/cb") || context.Request.Path == new Microsoft.AspNetCore.Http.PathString("/auth/cb/"))&&
+                                context.Request.Path == new PathString("/auth/cb") &&
```
Comment (defong) [inline src/ui/Startup.cs]: Magic string!!!

use 
```
options.CallbackPath = new PathString(Configuration["auth:oidc:callbackPath"]);
```

**Comment by defong** [comment]
On a different note can we extract the options into its own class altogether its becoming unworldly big

File: src/ui/Startup.cs (hunk 1 above)
Comment (the-daniel-rothig) [reply src/ui/Startup.cs]: Done :+1:

File: src/ui/Views/Shared/_Layout_base.cshtml [hunk 2]
```diff
@@ -11,19 +11,15 @@
 
     <link rel="stylesheet" href="~/application.css" asp-append-version="true" />
 
-    <!--[if lt IE 9]>
```
Comment (fofr) [inline src/ui/Views/Shared/_Layout_base.cshtml]: Do we not need the IE shiv? @dankmitchell

Comment (dankmitchell) [reply src/ui/Views/Shared/_Layout_base.cshtml]: That isn't working at present so best to remove. @the-daniel-rothig is going to create a card to add it properly.

**Pull request merged**